src/Ui/UiWebsocket.py

```diff
@@ -48,52 +49,21 @@ def start(self):
             file_server = sys.modules["main"].file_server
             if file_server.port_opened is None or file_server.tor_manager.start_onions is None:
                 self.site.page_requested = False  # Not ready yet, check next time
-            elif file_server.port_opened is True:
-                self.site.notifications.append([
-                    "done",
-                    _["Congratulation, your port <b>{0}</b> is opened.<br>You are full member of ZeroNet network!"].format(config.fileserver_port),
-                    10000
-                ])
-            elif config.tor == "always" and file_server.tor_manager.start_onions:
-                self.site.notifications.append([
-                    "done",
-                    _(u"""
-                    {_[Tor mode active, every connection using Onion route.]}<br>
-                    {_[Successfully started Tor onion hidden services.]}
-                    """),
-                    10000
-                ])
-            elif config.tor == "always" and file_server.tor_manager.start_onions is not False:
-                self.site.notifications.append([
-                    "error",
-                    _(u"""
-                    {_[Tor mode active, every connection using Onion route.]}<br>
-                    {_[Unable to start hidden services, please check your config.]}
-                    """),
-                    0
-                ])
-            elif file_server.port_opened is False and file_server.tor_manager.start_onions:
-                self.site.notifications.append([
-                    "done",
-                    _(u"""
-                    {_[Successfully started Tor onion hidden services.]}<br>
-                    {_[For faster connections open <b>{0}</b> port on your router.]}
-                    """).format(config.fileserver_port),
-                    10000
-                ])
             else:
-                self.site.notifications.append([
-                    "error",
-                    _(u"""
-                    {_[Your connection is restricted. Please, open <b>{0}</b> port on your router]}<br>
-                    {_[or configure Tor to become full member of ZeroNet network.]}
-                    """).format(config.fileserver_port),
-                    0
-                ])
+                try:
+                    self.addHomepageNotifications()
+                except Exception, err:
+                    self.log.error("Uncaught Exception: " + Debug.formatException(err))
 
         for notification in self.site.notifications:  # Send pending notification messages
+            # send via WebSocket
             self.cmd("notification", notification)
+            # just in case, log them to terminal
+            if notification[0] == "error":
+                self.log.error("\n\033[93mWARNING:\033[0m %s\n" % self.dedent(notification[1]))
```

Comment:
Adding some formatting/coloring to the output could be a good idea. But I think it has to done on the logging module level, because this way it messes up the output for non-compatible terminals and the log file.